performance_test/src/communication_abstractions/resource_manager.hpp

```diff
@@ -16,11 +16,15 @@
 #define COMMUNICATION_ABSTRACTIONS__RESOURCE_MANAGER_HPP_
 
 #include <rclcpp/rclcpp.hpp>
-#include <fastrtps/participant/Participant.h>
-#include <fastrtps/attributes/ParticipantAttributes.h>
-#include <fastrtps/Domain.h>
+
+#ifdef FASTRTPS_ENABLED
```

Comment:
As macros are global I would prefix them with `PERFORMANCE_TEST` as I did with most of the macros already. I forgot to do it with the `CONNEXT_DDS_MICRO_ENABLED`, so maybe you could change it there as well. Else this PR looks good to me!

Reply:
Yup, that makes sense. Also changed `MEMORY_TOOLS_ENABLED`, let me know if you'd prefer I revert that.

See e1077ae

Reply:
Thx!